CI Test Failure: Rate limit middleware unexpected 429 response

```
_____________________________ test_exclude_opt_key _____________________________

    def test_exclude_opt_key() -> None:
        @get("/excluded", skip_rate_limiting=True)
        def handler() -> None:
            return None
    
        @get("/not-excluded")
        def handler2() -> None:
            return None
    
        config = RateLimitConfig(rate_limit=("second", 1), exclude_opt_key="skip_rate_limiting")
    
        with create_test_client(route_handlers=[handler, handler2], middleware=[config.middleware]) as client:
            response = client.get("/excluded")
            assert response.status_code == HTTP_200_OK
    
            response = client.get("/excluded")
            assert response.status_code == HTTP_200_OK
    
            response = client.get("/not-excluded")
            assert response.status_code == HTTP_200_OK
    
            response = client.get("/not-excluded")
>           assert response.status_code == HTTP_429_TOO_MANY_REQUESTS
E           assert 200 == 429
E            +  where 200 = <Response [200 OK]>.status_code

tests/middleware/test_rate_limit_middleware.py:132: AssertionError
=========================== short test summary info ============================
FAILED tests/middleware/test_rate_limit_middleware.py::test_exclude_opt_key - assert 200 == 429
 +  where 200 = <Response [200 OK]>.status_code
============ 1 failed, 2078 passed, 5 skipped, 1 xfailed in 48.59s =============
Error: Process completed with exit code 1.
```

<!-- POLAR PLEDGE BADGE START -->
<a href="https://polar.sh/litestar-org/litestar/issues/1258">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar/issues/1258/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar/issues/1258/pledge.svg">
</picture>
</a>
<!-- POLAR PLEDGE BADGE END -->

--- a/tests/unit/test_middleware/test_rate_limit_middleware.py
+++ b/tests/unit/test_middleware/test_rate_limit_middleware.py
@@ -120,6 +120,7 @@ async def test_reset() -> None:
         assert response.status_code == HTTP_200_OK
 
 
+@travel(datetime.utcnow, tick=False)
 def test_exclude_patterns() -> None:
     @get("/excluded")
     def handler() -> None:
@@ -145,6 +146,7 @@ def test_exclude_patterns() -> None:
         assert response.status_code == HTTP_429_TOO_MANY_REQUESTS
 
 
+@travel(datetime.utcnow, tick=False)
 def test_exclude_opt_key() -> None:
     @get("/excluded", skip_rate_limiting=True)
     def handler() -> None:
@@ -170,6 +172,7 @@ def test_exclude_opt_key() -> None:
         assert response.status_code == HTTP_429_TOO_MANY_REQUESTS
 
 
+@travel(datetime.utcnow, tick=False)
 def test_check_throttle_handler() -> None:
     @get("/path1")
     def handler1() -> None:
@@ -198,6 +201,7 @@ def test_check_throttle_handler() -> None:
         assert response.status_code == HTTP_200_OK
 
 
+@travel(datetime.utcnow, tick=False)
 async def test_rate_limiting_works_with_mounted_apps(tmpdir: "Path") -> None:
     # https://github.com/litestar-org/litestar/issues/781
     @get("/not-excluded")

